tests/components/jewish_calendar/test_binary_sensor.py

```diff
@@ -90,7 +92,5 @@
         async_fire_time_changed(hass, future)
         await hass.async_block_till_done()
 
-        assert (
-            hass.states.get("binary_sensor.test_issur_melacha_in_effect").state
-            == result
-        )
+        assert (hass.states.get(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)